woocommerce/woocommerce
# Fix 'dynamic property declaration' warnings in wc-admin code (PHP 8.2+)

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Fixes PHP `Deprecated: Creation of dynamic property warnings` in wc-admin-related code when running on PHP 8.2+.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure your environment is running PHP 8.2.x and has logging enabled.
2. Go to the frontend and add an item to the cart.
3. Go to the checkout page.
5. Use grep (`grep 'WooCommerce\\Admin' logfile.log`) or manual inspection to confirm that:
   - On `trunk`, a few warnings from wc-admin are present.
   - On this branch, those warnings no longer appear.
